mesonbuild/interpreter/kwargs.py

```diff
@@ -1,10 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright © 2021 The Meson Developers
 # Copyright © 2021 Intel Corporation
-from __future__ import annotations
 
 """Keyword Argument type annotations."""
 
+# We very intentionally don't use `__future__.annotations` here, as it is
+# unnecessary and confues flake8
```

Comment:
I agree that it's unnecessary -- it was added as part of a treewide insert operation (commit 680b5ff819bf36397ea560f802510bce5b83f6bc, my fault).

That being said, it should be harmless, and it doesn't cause any flake8 errors for me?

Reply:
(Also the assert T.TYPE_CHECKING seems like flake8 should not care.)

Reply:
I can't remember exactly when I ran into the errors, somewhere along the giant build refactor series. Somewhere along the lines in that series having `__future__.annotations` confuses the heck out of flake8. Since it doesn't seem to be a problem here I'll just drop it from this series

Reply:
Yeah, the assert was more an attempt to document/enforce that module shouldn't be imported in live code